### PR TITLE
fluidsynth: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
2.3.3

### CODE REVIEW

**Comment by Managor** [comment]
There is probably a lot more to this command but I've only ever used it as a toy so I put only the most simplest form there.

File: pages/linux/fluidsynth.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# fluidsynth
+
+> Synthesize audio from midi files.
+> More information: <https://www.fluidsynth.org/documentation/>.
```
Comment (kbdharun) [inline pages/linux/fluidsynth.md:4]:
```suggestion
> More information: <https://github.com/FluidSynth/fluidsynth/wiki/UserManual>.
```
I think it is better to link to the user manual directly.

File: pages/linux/fluidsynth.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# fluidsynth
+
+> Synthesize audio from midi files.
```
Comment (kbdharun) [inline pages/linux/fluidsynth.md:3]:
```suggestion
> Synthesize audio from MIDI files.
```

File: pages/linux/fluidsynth.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://www.fluidsynth.org/documentation/>.
+
+- Play a midi file:
+
+`fluidsynth --audio-driver={{pipewire|pulseaudio}} {{path/to/soundfont}}.sf2 {{path/to/midi}}.midi`
```
Comment (kbdharun) [inline pages/linux/fluidsynth.md:8]:
```suggestion
`fluidsynth --audio-driver={{pipewire|pulseaudio}} {{path/to/soundfont}}.sf2 {{path/to/file}}.midi`
```

File: pages/linux/fluidsynth.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# fluidsynth
+
+> Synthesize audio from midi files.
+> More information: <https://www.fluidsynth.org/documentation/>.
+
+- Play a midi file:
```
Comment (kbdharun) [inline pages/linux/fluidsynth.md:6]:
```suggestion
- Play a MIDI file:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

File: pages/linux/fluidsynth.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# fluidsynth
+
+> Synthesize audio from MIDI files.
+> More information: <https://github.com/FluidSynth/fluidsynth/wiki/UserManual>.
+
+- Play a MIDI file:
+
+`fluidsynth --audio-driver={{pipewire|pulseaudio}} {{path/to/soundfont}}.sf2 {{path/to/file}}.midi`
```
Comment (acuteenvy) [inline pages/linux/fluidsynth.md:8]:
```suggestion
`fluidsynth --audio-driver={{pipewire|pulseaudio}} {{path/to/soundfont.sf2}} {{path/to/file.midi}}`
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**